hub/core/dataset/dataset.py

```diff
@@ -1138,7 +1142,57 @@ def delete(self, large_ok=False):
         self._unlock()
         self.storage.clear()
 
+    def summary(self):
+        head = [
+            "tensor",
+            "htype",
+            "shape",
+            "dtype",
+            "compression",
+        ]  # To add more attributes change head, divider, row_array and table_str
+        divider = ["-------"] * 5
+        tensor_dict = self.version_state[
+            "full_tensors"
+        ]  # Creating a list of tensors in the dataset
```

Comment:
Comment not required - this is a dict not a list, and should `self.tensors` instead of `self.version_state["full_tensors"]`